### DISCUSSION TITLE
Texture duplication?

### DISCUSSION THREAD

**Discussion by Tiggs62** [kubilus1/autoortho#392] [General]
![b738 - 2023-09-19 09 21 44](https://github.com/kubilus1/autoortho/assets/54512432/442d5aea-8853-4bc0-88f6-db36282ca741)

Hi. Been seeing this a few tiles lately, whre it appears the runway texture image is repeated in blocks. Its only at ground/low altitude, if I go higher it vanishes. 

0.6.0-1

**Comment by Jeff2122** [comment]
Install v0.5.1-3 and your problem will go away. It did on my system and I hope it helps you too. I have tried all versions from 0.4.4 to v0.6.1 and have come to the conclusion that v0.6.x is broken. 
v0.5.1-3 uses the same data so no need to reinstall. You just have to be careful to never open XP without the server or you will be doing the ini dance. That is the one thing that the new versions do well is maintain the ini files better.
v0.5.1-3 has the advantage of using the new scenery folder arrangement (much much cleaner) but doesn't have the tile problem of v0.6.x. A change was made to the latest version that is presenting bad tiles too often.

**Reply by Tiggs62** [reply]
Ah, that does look better
